test

```diff
@@ -57,7 +57,16 @@ function unit_tests {
 }
 
 function integration_tests {
-	echo "Running integration tests..."
+	MIGRATION_TEST="v3.0.2"
```

Comment:
this should probably get the last release version from git (grep for `version: bump`?)